tasks/libs/common/omnibus.py

```diff
@@ -1,9 +1,164 @@
+import hashlib
 import json
 import os
 import sys
 from datetime import datetime
 
 import requests
+from release import _get_release_json_value
+
+
+def _get_build_images(ctx):
+    # We intentionally include both build images & their test suffixes in the pattern
+    # as a test image and the merged version shouldn't share their cache
+    tags = ctx.run("grep -E 'DATADOG_AGENT_.*BUILDIMAGES' .gitlab-ci.yml | cut -d ':' -f 2", hide='stdout').stdout
+    return (t.strip() for t in tags.splitlines())
+
+
+def _get_omnibus_commits(field):
+    release_version = os.environ['RELEASE_VERSION_7']
```

Comment:
I think you want to check `RELEASE_VERSION`, and then  `RELEASE_VERSION_7` if the first one is not found, because of the way we set these variables currently.

Note for later: we should standardize all builds, make them set `RELEASE_VERSION` explicitly, to avoid having to use this hack.

Reply:
Indeed this was causing a failure on windows.
I might have some questions about these variables during the summit, it's a bit unclear to me.